### DISCUSSION TITLE
Questions about Astroid framework plugin and best time to ask for support

### DISCUSSION THREAD

**Discussion by DimasPLT** [templaza/astroid-framework#698] [Q&A]
Hello,

I have a couple of questions regarding the Astroid framework.

How often do the developers of the Astroid framework respond to questions or issues raised by users?
Is there a preferred time or day when it’s best to post questions to get a quicker response?
Additionally, I encountered an issue today where I was not aware that I needed to download an additional plugin. In the 'Assets' section, there are only 'astroid-framework-3.0.15.zip', 'astroid-package-v3.0.15.Framework-Template.zip', 'astroid-quickstart.zip', and 'source code'. This information is not mentioned in the 'How to install' section.

I believe adding this information to the documentation could help newcomers avoid confusion.

Thank you for your assistance!

**Comment by sonvnn** [comment]
Thank you for your feedback. I will improve it. 

For the support time. You can post your question anytime because We are an international community spanning across the globe. Because we work as volunteers, the answer may come depending on their free time.

**Comment by Chacapamac** [comment]
> where I was not aware that I needed to download an additional plugin
Can somebody explain what this is about? “Need to install a plugin” to Install Astroid?
